server/src/test/java/org/cloudfoundry/identity/uaa/zone/MultitenantJdbcClientDetailsServiceTests.java

```diff
@@ -294,6 +295,7 @@ void loadingClientIdWithSingleDetails() {
         jdbcTemplate.update(INSERT_SQL,
                 "clientIdWithSingleDetails",
                 "mySecret",
+                "myClientJwtConfig",
```

Comment:
Please cast the result in line 308 to `UaaClientDetails` and then check whether the value is set correctly

Reply:
done